docs/content/concepts/assets/multi-assets.mdx

```diff
@@ -46,6 +46,39 @@ def my_function():
 
 By default, the names of the outputs will be used to form the asset keys of the multi-asset. The decorated function will be used to create the op for these assets and must emit an output for each of them. In this case, we can emit multiple outputs by returning a tuple of values, one for each asset.
 
+### Conditional materialization
+
+In some cases, an asset may not need to be updated in storage each time the decorated function is executed. You can use the `is_required` parameter along with `yield` syntax to invoke this behavior. If the `is_required` parameter is set to `False` on an output, you may indicate to the Dagster framework that no data should be persisted to storage by not yielding an output from your computation function. If an output is not emitted during computation, no asset materialization event will be created, the I/O manager will not be invoked, downstream assets will not be materialized, and asset sensors monitoring the asset will not trigger.
```

Comment:
> If the `is_required` parameter is set to `False` on an output, you may indicate to the Dagster framework

This feels a bit odd to me - a few thoughts:

- Do we 'indicate' things to Dagster? 
- Is it Dagster or the Dagster framework?

Would something like this still get the point across?

> Setting the `is_required` parameter to `False` on an output tells Dagster that no data should be persisted to storage by not yielding an output from the computation function.

Reply:
FWIW this is copy/pasted from the analogous section in the assets page. I agree that there are opportunities to tighten it up though.

> Would something like this still get the point across?

Not 100%, because "not yielding an output from your computation function" is what tells Dagster that no data should be persisted to storage, not just "Setting the is_required parameter to False".

I made a similar change based on your recommendation. Let me know what you think.